frontend/src/app/containers/ExperimentPage/DetailsDescription.js

```diff
@@ -85,11 +85,11 @@ export default function DetailsDescription({
                     </strong>
                   </Localized>}
                 {copy &&
-                  <Localized id={l10nId(["details", idx, "copy"])}>
+                  <LocalizedHtml id={l10nId(["details", idx, "copy"])}>
```

Comment:
None of the current experiments use markup for the details copy. It would be safer to use a regular `<Localized>` here.